sdk/communication/azure-communication-callautomation/src/test/java/com/azure/communication/callautomation/CallConnectionAsyncUnitTests.java

```diff
@@ -289,4 +300,233 @@ public void removeParticipantsWithResponse() {
         assertNotNull(repeatabilityHeaders.getRepeatabilityFirstSentInHttpDateFormat());
         assertNotNull(repeatabilityHeaders.getRepeatabilityRequestId().toString());
     }
+
+    @Test
+    public void muteParticipant() {
+        CallConnectionAsync callConnectionAsync = getCallAutomationAsyncClient(new ArrayList<SimpleEntry<String, Integer>>(
+            Arrays.asList(
+                new SimpleEntry<String, Integer>(serializeObject(new MuteParticipantsResponseInternal()), 202)
+            )))
+            .getCallConnectionAsync(CALL_CONNECTION_ID);
+
```

Comment:
General question: didn't find any session records update, but the tests are updated

Reply:
I added and updated unit tests.
Unit tests don't need session records, but the live tests do.